src/lib/validation/documentation.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from "path";
+import * as ts from "typescript";
+import { ProjectReflection, ReflectionKind } from "../models";
+import { Logger, normalizePath } from "../utils";
+
+export function validateDocumentation(
+    project: ProjectReflection,
+    logger: Logger,
+    requiredToBeDocumented: readonly (keyof typeof ReflectionKind)[]
+): void {
+    const kinds = requiredToBeDocumented.reduce(
+        (prev, cur) => (prev |= ReflectionKind[cur]),
+        0
+    );
+
+    for (const ref of project.getReflectionsByKind(kinds)) {
+        const symbol = project.getSymbolFromReflection(ref);
+        if (!ref.comment && symbol?.declarations) {
+            const decl = symbol.declarations[0];
+            const sourceFile = decl.getSourceFile();
+            const { line } = ts.getLineAndCharacterOfPosition(
```

Comment:
There's an off by one error here, getLineAndCharacterOfPosition returns a 0 based line, but we should display a 1 based line

Reply:
Ah thanks, will fix.